### ISSUE TITLE
Test analyses that have 3+ group levels

### ISSUE THREAD

**Issue by WillemSleegers** [gasparl/tidystats-Google-Docs-add-in#3]
Because the sidebar in a Google add-in is fixed at 300px, this might be an issue for analyses that have many groups within groups. We should test what that looks like and figure out a way to work with that limit.

**Comment by gasparl** [comment]
Looks fine to me. Even if there are many groups, they are all easily visible, and if some text is too long, a ~~vertical~~ horizontal scroll appears so that you can scroll rightwards to see everything. (See e.g. with [this](https://textdoc.co/wGkqnKLuoaWxciR5) test data.)

**Comment by WillemSleegers** [comment]
Yeah that's what I was afraid of. Horizontal scroll bars are not allowed to happen.

**Comment by gasparl** [comment]
Sorry I of course meant horizontal scroll. Both vertical and horizontal are working fine.

**Comment by WillemSleegers** [comment]
I know, but they shouldn't. =p A vertical scroll bar is fine, but horizontal bars are not. It looks bad and it's not pleasant to have to scroll sideways. I'll probably work on fixing the naming issue, but then there is still the issue of certain analyses having many levels of depth.
